packages/langium/test/languages/domain-model/lsp/goto-resolver.test.ts

```diff
@@ -0,0 +1,76 @@
+/******************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { Diagnostic, Position } from 'vscode-languageserver';
+import { buildRealpath, createLangiumGrammarServices, LangiumDocument, LangiumDocumentConfiguration, LangiumGrammarServices, loadUriContent } from '../../../../src';
+
+let services: LangiumGrammarServices;
+let document: LangiumDocument;
+
+beforeAll(() => {
+    const realPathFile = buildRealpath('packages/langium', 'test/languages/domain-model/domain-model.langium');
```

Comment:
Why not use `path.join(__dirname, '..', 'domain-model.langium')`?

Reply:
Because I didn't know `__dirname` is available. This basically resolves the need for the file-utils. It will be removed. Already tested this locally.